nautobot/docs/release-notes/version-2.0.md

```diff
@@ -144,6 +144,24 @@ Exporting objects and lists of objects to CSV format has been totally reimplemen
 
 Conversely, importing objects from CSV format has also been reimplemented in the same new framework. The REST API can now accept CSV files as well as the existing JSON support, and the UI for importing CSVs uses this same framework behind the scenes.
 
+In addition to the above improvements, you can now reference related objects in your CSV by using a combination of unique fields. For instance:
+
+Instead of:
+
+```csv
+name,rack
+Device one,7f3ca431-8103-45cc-a9ce-b94c1f784a1d
+```
+
+you can use:
+
+```csv
+name,rack__location__name,rack__name
+Device one,Equinix DC6,R204
+```
+
+This enhancement allows you to specify related objects using their unique attributes, making data import even more intuitive and flexible.
+
 An immediate benefit you can notice from this reimplementation is that CSVs should now **generally** be "round-trip" capable, meaning that you can export a set of records to CSV format and then import that CSV into a different Nautobot instance (or delete the records and use the CSV to recreate them) without needing to "massage" the CSV into a different set of columns or fields.
```

Comment:
I'd suggest swapping the order here as I think it flows better - the multi-column feature doesn't really apply to the round-trip functionality.

````suggestion
An immediate benefit you can notice from this reimplementation is that CSVs should now **generally** be "round-trip" capable, meaning that you can export a set of records to CSV format and then import that CSV into a different Nautobot instance (or delete the records and use the CSV to recreate them) without needing to "massage" the CSV into a different set of columns or fields.

In addition to the above improvements, you can now reference related objects in your CSV by using a combination of unique fields. For instance:

Instead of:

```csv
name,rack
Device one,7f3ca431-8103-45cc-a9ce-b94c1f784a1d
```

you can use:

```csv
name,rack__location__name,rack__name
Device one,Equinix DC6,R204
```

This enhancement allows you to specify related objects using their unique attributes, making data import even more intuitive and flexible.
````